packages/components/src/unit-control/index.tsx

```diff
@@ -168,6 +171,50 @@ function UnforwardedUnitControl(
 				refInputSuffix.current?.focus();
 			}
 		};
+		onPaste = ( event: ClipboardEvent< HTMLInputElement > ) => {
+			props.onPaste?.( event );
```

Comment:
Just a thought that I don't think we never discussed before — should we call `props.onPaste` at the start or at the end of the internal callback? The same reasoning should probably be applied to other callbacks (e.g. `onChange`)